ganglia/ganglia_api.py

```diff
@@ -74,7 +74,7 @@ def api_dict(self):
                   'description': self.desc,
                   'sum': self.sum,
                   'num': self.num,
-                  'value': ApiMetric.is_num(self.val),
+                  'value': ApiMetric.is_num(self.val, self.type),
```

Comment:
why are we passing the type in and not using it? I agree this should change to be a function with this signature (though not named `is_num`) but since we aren't using it in the function can we avoid passing it here for now?

Reply:
Probably because I changed the is_num logic to match what was already there, and removed the code that used type == "string".  I'll fix that...

Reply:
also because i didn't run my source tools since they're on another branch that hasn't been merged yet.  :(